comment/api/serializers.py

```diff
@@ -89,6 +89,15 @@ def get_dislikes(obj):
         if hasattr(obj, 'dislikes'):
             return obj.dislikes
 
+    @staticmethod
+    def get_is_flagged(obj):
+        if hasattr(obj, 'flag'):
```

Comment:
The object already has `is_flagged` property just return this property instead of repeating the same method.
```python
return obj.is_flagged
```

Reply:
Previous comments will not have a `flag` object.

Reply:
I am still seeing the old code :) I will resolve it once done.